### DISCUSSION TITLE
v8.2 (fixed issue)

### DISCUSSION THREAD

**Discussion by anothermartz** [anothermartz/Easy-Wav2Lip#32] [Announcements]
I fixed issue #28 which broke the colab version completely and may have prevented new local installs from working. So now things should work as they did before 🎉

[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/anothermartz/Easy-Wav2Lip/blob/v8.2/Easy_Wav2Lip_v8.2.ipynb)
https://colab.research.google.com/github/anothermartz/Easy-Wav2Lip/blob/v8.2/Easy_Wav2Lip_v8.2.ipynb

**For Easy-Wav2Lip.bat users:**
It should prompt you to update to 8.2 (I've included Easy-Wav2Lip.bat here for new users but there are no changes to the last version).

For manual install users just pull from the v8.2 branch and run install.py

### How did it break?
A module called torchvision updated and with it changed how some of its code is called. Another module called basicsr was still using the old way and as a result, broke when gfpgan tried to install. This kills the step 1. 🦀

To fix this I added the offending file from basicsr into my project and fixed the outdated line. First I install basicsr, then I replace the old file with my fixed file. Now when gfpgan installs, it doesn't get upset and burn the the entire project install :)

### 200 Stars!?
Also I've realised that this project is slowly gaining traction when I thought it wasn't really being used so I will try to work on a demonstration video, update the readme with pictures and work on improving the usability and result quality.

<hr /><em>This discussion was created from the release <a href='https://github.com/anothermartz/Easy-Wav2Lip/releases/tag/v8.2'>v8.2 (fixed issue)</a>.</em>

**Comment by kenist0n** [comment]
Thanks for the quick fix. Also really great script, just wanted to leave that here :-)

**Comment by Mishmallow11** [comment]
Thank you very much, buddy!
Everything works perfectly. Simply great!

**Comment by GreenRaptor11** [comment]
Good Afternoon! The program is functioning well, but slower than usual. Additionally, the downloaded file with the results turns out to be corrupted.

**Reply by anothermartz** [reply]
> Good Afternoon! The program is functioning well, but slower than usual. Additionally, the downloaded file with the results turns out to be corrupted.

Interesting, could you please open a new thread in the issues section about this?

It will be easier to discuss and allows other people experiencing the same to find it and give info.
